Login.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+
+function Login() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleChange = (event) => {
+    const { name, value } = event.target;
+
+    if (name === 'email') {
+      setEmail(value);
+    } else if (name === 'password') {
```

Comment:
Consider using a single state object to handle form data instead of having separate state variables for each field. This will make the code cleaner and easier to maintain.